### DISCUSSION TITLE
The links should be highlighted, to look like a link.

### DISCUSSION THREAD

**Discussion by mishainflux** [GitbookIO/community#670] [Feature Requests]
Once any link is added to any page/space, the colour of the link shall change to maybe a blue link to showcase that it is a link. The colour of the link stays black in the editable app(without any option to change it), 
![editable page](https://github.com/user-attachments/assets/cda10fc6-b694-44b2-adc9-a0c9a0c5a736)
but in the gitbook, it stays black but underlined for viewers. They see it's a link once the user hovers their cursor onto the underlined text. It would be great if there would be an automatic different colour for links.
![same page on gitbook for viewers](https://github.com/user-attachments/assets/ce9912ce-3021-43d4-907b-ddbd9cc97c91)

**Comment by SamyPesse** [comment]
@mishainflux What is the URL of your public content? Links are usually colored with the primary color you configured.

In the application (first screenshot), we'll iterate on those.

**Reply by mishainflux** [reply]
@SamyPesse Hello Samy, Saw the recording of your webinar; it was great. Here is the link for our docs page on gitbook. https://docs.influxtechnology.com/support_guides 
I had to change the colour of each one to blue using the editing bar. I was hoping at least the links would be highlighted in blue (or any other colour) to make them stand apart. Also, as the editing bar can not be accessed for links, I could not change the colour manually for them.
